api/responses/serverError.js

```diff
@@ -1,11 +1,12 @@
 const { logger } = require('../../winston');
 
 module.exports = (error = {}, { res }) => {
-  logger.error('Sending 500: %o', error);
+  // TODO - remove when refactoring error handling/logging
+  logger.error('Sending 500: ', error);
```

Comment:
The `%o` didn't seem to be doing anything and now the stack trace is logged, yay